tasks/main.yml

```diff
@@ -17,6 +17,18 @@
     state: "{{ docker_service_state }}"
     enabled: "{{ docker_service_enabled }}"
 
+- name: Ensure /etc/docker/ directory exists.
+  file:
+    path: /etc/docker
+    state: directory
```

Comment:
The last thing I'd like to make sure to do is _not_ touch the file if the default (`docker_daemon_options: {}`) is set.

That way (a) existing role users won't have that file wiped out if it currently contains things and they update the role, and (b) people can manage the file on their own simply by _not_ setting the option in this role's vars.